electionleaflets/settings/base.py

```diff
@@ -110,6 +111,13 @@
     'django_extensions',
 ] + LEAFLET_APPS
 
+if environ.get('SENTRY_DSN', None):
+    INSTALLED_APPS.append('raven.contrib.django.raven_compat')
+    RAVEN_CONFIG = {
+        'dsn': environ.get('SENTRY_DSN'),
+        'environment': environ.get('SENTRY_ENVIRONMENT', 'production')
```

Comment:
Out of interest, where is the `'environment'` config documented? I couldn't find it here: https://docs.sentry.io/clients/python/integrations/django/